docs/index.rst

```diff
@@ -1,12 +1,18 @@
+.. image:: images/IQM_Logo_RGB_frame.jpg
+    :width: 800
+
 Cirq on IQM
-###########
+===========
 
 :Release: |release|
 :Date: |today|
-
+:Source Code: `<https://github.com/iqm-finland/cirq-on-iqm>`_
 
 .. include:: ../README.rst
-   :start-line: 7
+    :start-line: 9
+    :end-before: Installation
```

Comment:
This is so specific it'll break when someone edits README :)
Maybe it's best just to copypaste the text here.

Reply:
The best solution would be to have a separate `.rst` file containing the introduction and include it in both `README.rst` and `index.rst`, but unfortunately GitHub does not allow importing in the README. Nevertheless, duplicating the text seems to me less harmful than including like this )).

Reply:
Copied the text and removed the include statement